src/players/Player.js

```diff
@@ -0,0 +1,47 @@
+import { MESSAGE } from '../constants/constants.js';
+import View from '../view/View.js';
+
+export default class Player {
+
+  async inputNumber() {
+    const userInput = await View.readInput(MESSAGE.input);
+
+    if (Player.isValidNumber(userInput)) {
+      this._playerNumber = userInput.split('').map(num => parseInt(num, 10));
+    }
+  }
+
+  getJudgeResult(computer) {
+    const result = computer.judgeResult(this._playerNumber);
+    return result;
+  }
+
+  static isValidNumber(userInput) {
+    const userInputToArray = userInput.split('');
+
+    if (userInputToArray.length !== 3) {
+      throw new Error(MESSAGE.error);
+    }
+
+    if (userInputToArray.some(char => isNaN(parseInt(char, 10)))) {
```

Comment:
전역 `isNaN`은 숫자가 아닌 것을 숫자로 강제하고, NaN으로 간주되는 모든 것을 true로 반환합니다.
`Number.isNaN`을 사용하면 의도치 않은 결과를 방지할 수 있습니다.

```suggestion
    if (userInputToArray.some(char => Number.isNaN(parseInt(char, 10)))) {
```

[참고 자료]
https://github.com/parksb/javascript-style-guide#standard-library--isnan

Reply:
아 이런 내용도 있었네요
꼼꼼한 리뷰 감사합니다 ㅎㅎ